pkg/postage/listener/listener_test.go

```diff
@@ -379,10 +379,6 @@ func TestListenerBatchState(t *testing.T) {
 				noOfEvents++
 				switch ev := e.(type) {
 				case blockNumberCall:
-					if ev.blockNumber < 497 && ev.blockNumber > 500 {
-						errs <- fmt.Errorf("invalid blocknumber call %d", ev.blockNumber)
-						return
-					}
```

Comment:
why this block got removed?

Reply:
Please, see my comment on this PR below.